### PR TITLE
Revisit README

### PR DESCRIPTION
**Pull request by zdne**
For discussion. Will squash before merge. 

I have revisited, updated and hopefully simplified the README. All the developer related information to be pushed outside of README. 

I have also removed the TLDR section. 

```
## TL;DR
- API design language
- Plain text
- Built for humans
- Understandable by machines
- Complete API Design lifecycle
- Open & Free
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,98 +5,119 @@
 
 [![slack](https://apiblueprint-slack.herokuapp.com/badge.svg)](http://apiblueprint-slack.herokuapp.com/)
 
-API Blueprint is a documentation-oriented API description language. A couple of semantic assumptions over the plain [Markdown](http://daringfireball.net/projects/markdown/).
+A powerful high-level API design language for web APIs.
 
-API Blueprint is perfect for designing your Web API and its comprehensive documentation but also for quick prototyping and collaboration. It is easy to learn and even easier to read – after all it is just a form of plain text.
+API Blueprint is simple and accessible to everybody involved in the API design
```
Comment (kylef) [inline README.md]:
```diff
- Its syntax is concise yet expressive
+ It's syntax is concise yet expressive
```

File: README.md [hunk 2]
```diff
@@ -5,98 +5,119 @@
...
+[mock]: http://docs.apibstart.apiary.io/#reference/0/message/get?console=1
+[documentation]: http://docs.apibstart.apiary.io
+[test]: https://github.com/apiaryio/dredd
+[tutorial]: Tutorial.md
+[examples]: https://github.com/apiaryio/api-blueprint/tree/master/examples
 
-If you are interested in building tools for API Blueprint or just to integrate it with your workflow check out the [Developing tools for API Blueprint](https://github.com/apiaryio/api-blueprint/wiki/Developing-tools-for-API-Blueprint).
+## Version
+- Current version: [1A9](https://github.com/apiaryio/api-blueprint/releases/tag/format-1A9)
+- Version supported by Apiary: [1A8](https://github.com/apiaryio/api-blueprint/releases/tag/format-1A8)
```
Comment (pksunkara) [inline README.md]: `1A9` is supported by Apiary.

By the way, why doesn't MSON have versions? MSON `nullable` is considered current version but it is not yet supported in Apiary.

File: README.md [hunk 3]
```diff
@@ -5,98 +5,119 @@
...
+A powerful high-level API design language for web APIs.
 
-API Blueprint is perfect for designing your Web API and its comprehensive documentation but also for quick prototyping and collaboration. It is easy to learn and even easier to read – after all it is just a form of plain text.
+API Blueprint is simple and accessible to everybody involved in the API design
+lifecycle. Its syntax is concise yet expressive.
 
-API Blueprint, its parser, and most of its tools are completely open sourced so you don't have to worry about vendor lock-in. This also means you can freely integrate API Blueprint into any type of product, commercial or not.
+With API Blueprint you can quickly prototype and model APIs to be created or
+describe already deployed mission-critical APIs. From a [car][tesla] to the
+largest CDN in the world.
```
Comment (kylef) [inline README.md]: I'd perhaps use "Content Distribution Network (CDN)" for those un-aware of what a CDN is, I can't imagine it being too widely used within the API community.

**Comment by pksunkara** [comment]
Looks good.

File: README.md [hunk 4]
```diff
@@ -5,98 +5,119 @@
...
 
-## TL;DR
-+ Web API description language
-+ Pure Markdown
-+ Designed for humans
-+ Understandable by machines
-+ Open & free
+The API Blueprint is built to encourage dialog and collaboration between project
```
Comment (kylef) [inline README.md]: Should this be `dialogue` not `dialog`, or is this equivalent in :us: English? I though dialog was one of these:

iu-1

File: README.md [hunk 5]
```diff
@@ -5,98 +5,119 @@
...
+[test][] for your API before you even start coding.
 
-\[[full listing](https://github.com/apiaryio/api-blueprint-ast#json-serialization)\]
+To learn more about the API Blueprint syntax jump directly to the
+[API Blueprint Tutorial][tutorial] or take a look at some [examples][].
 
-It is the task for the API Blueprint parser – [Drafter][] or one of its language bindings to translate the API Blueprint Markdown representation into a machine friendly format – AST.
+[mock]: http://docs.apibstart.apiary.io/#reference/0/message/get?console=1
+[documentation]: http://docs.apibstart.apiary.io
+[test]: https://github.com/apiaryio/dredd
```
Comment (kylef) [inline README.md]: Perhaps this should link to dredd documentation instead: http://dredd.readthedocs.org/en/latest/

File: README.md [hunk 6]
```diff
@@ -5,98 +5,119 @@
...
+[tutorial]: Tutorial.md
+[examples]: https://github.com/apiaryio/api-blueprint/tree/master/examples
 
-If you are interested in building tools for API Blueprint or just to integrate it with your workflow check out the [Developing tools for API Blueprint](https://github.com/apiaryio/api-blueprint/wiki/Developing-tools-for-API-Blueprint).
+## Version
+- Current version: [1A9](https://github.com/apiaryio/api-blueprint/releases/tag/format-1A9)
```
Comment (kylef) [inline README.md]: Off-topic, should we have released 1A10 with MSON nullability? Or would MSON features not warrant API Blueprint updates?

Comment (pksunkara) [inline README.md]: This is something I just realised and we need to figure it out. https://github.com/apiaryio/api-blueprint/pull/292#discussion_r47488338

File: README.md [hunk 7]
```diff
@@ -5,98 +5,119 @@
...
-## License
-MIT License. See the [LICENSE](https://github.com/apiaryio/api-blueprint/blob/master/LICENSE) file.
+[issues]: https://github.com/apiaryio/api-blueprint/issues
+[rfc]: https://github.com/apiaryio/api-blueprint-rfcs
 
-[Drafter]: https://github.com/apiaryio/drafter
-[RFCs]: https://github.com/apiaryio/api-blueprint-rfcs
+## Get in Touch
+- [@apiblueprint](https://twitter.com/apiblueprint)
+- [Slack](http://apiblueprint-slack.herokuapp.com/)
```
Comment (kylef) [inline README.md]: Let's use `https://` :smile:.

File: README.md (hunk 2 above)
Comment (zdne) [inline README.md]: Good catch @pksunkara ! I am actually going to remove this section completely, I would like to have the version supported in Apiary listed only in the apiary documentation ..

File: README.md (hunk 3 above)
Comment (zdne) [inline README.md]: OK. (Another higher grade example would be banking API ... )

File: README.md (hunk 1 above)
Comment (kylef) [inline README.md]: Actually, I'm wrong here. Leave it as it is.

File: README.md (hunk 4 above)
Comment (zdne) [inline README.md]: Interesting. Will change it to "dialogue". I guess it was my simplified english :)

http://english.stackexchange.com/questions/59640/what-is-the-difference-between-dialogue-and-dialog

File: README.md (hunk 6 above)
Comment (zdne) [inline README.md]: I think versioning and the relation of MSON and API Blueprint is something we have to figure out. TBD. Meanwhile I am removing this section.

File: README.md (hunk 7 above)
Comment (zdne) [inline README.md]: :+1:

**Comment by zdne** [comment]
@kylef updated for review comments

**Comment by zdne** [comment]
Going to squash the commits

**Comment by zdne** [comment]
@kylef squashed and good to merge (for me)

**Pull request merged**